jogo42-cb/main.c

```diff
@@ -18,6 +18,16 @@ void MoverJog(GameState* gs);
 // Move os obstaculos
 void MoverObst(GameState* gs);
 
+const float dis=100.0;
+
+bool ataque (GameState* gs)
+{
+    Vector2 Atk;
+    Atk.x= gs->jog.pos.x + dis*cos(gs->jog.rot/60);
+    Atk.y= gs->jog.pos.y + dis*sin(gs->jog.rot/60);
+    return CheckCollisionCircles(Atk, 10, gs->obstCircCentro, gs->obstCircRaio);
```

Comment:
O raio da hitbox de ataque (`10`) deveria ser definido por uma constante (definida em `jogo42.h`).

Um valor que é colocado assim sem explicação do que ele significa é chamado de "[número mágico](https://en.wikipedia.org/wiki/Magic_number_(programming))", e é uma prática ruim porque deixa o código mais difícil de entender. Além disso, se esse valor for usado várias vezes é bem pior, porque aí se for mudar tem que mudar em vários lugares (e a pessoa pode se esquecer de algum, o que pode gerar um bug bem difícil de achar).